core/src/main/java/com/linecorp/armeria/client/UnprocessedRequestException.java

```diff
@@ -29,19 +34,31 @@
 
     private static final long serialVersionUID = 4679512839715213302L;
 
-    private static final UnprocessedRequestException INSTANCE = new UnprocessedRequestException(false);
+    /**
+     * Creates a new instance with the specified {@code message} and {@code cause}.
+     */
+    public UnprocessedRequestException(@Nullable String message, Throwable cause) {
+        super(message, requireNonNull(cause, "cause"));
+    }
 
     /**
-     * Returns an {@link UnprocessedRequestException} which may be a singleton or a new instance, depending on
-     * whether {@linkplain Flags#verboseExceptions() the verbose exception mode} is enabled.
+     * Creates a new instance with the specified {@code cause}.
      */
-    public static UnprocessedRequestException get() {
-        return Flags.verboseExceptions() ? new UnprocessedRequestException() : INSTANCE;
+    public UnprocessedRequestException(Throwable cause) {
+        super(requireNonNull(cause, "cause"));
     }
 
-    private UnprocessedRequestException() {}
+    @Nonnull
+    @Override
+    public synchronized Throwable getCause() {
```

Comment:
`synchronized` is unnecessary because `super.getCause()` is already synchronized, i.e.

```java
public Throwable getCause() {
    synchronized (this) {
        // return super.getCause() is actually:
        synchronized (this) {
            return (cause==this ? null : cause)
        }
    }
}
```

Reply:
I didn't think of that. Thanks.